package/examples/contacts/conf_protein.gro

```diff
@@ -0,0 +1,929 @@
+Generated by trjconv : topol.py t=   0.00000
```

Comment:
add in bzipped form --- we can handle it transparently and saves about 80% space